src/gluonts/distribution/box_cox_tranform.py

```diff
@@ -254,10 +254,10 @@ def __init__(self, lb_obs: float = 0.0, fix_lambda_2: bool = True) -> None:
     def domain_map(self, F, *args: Tensor) -> Tuple[Tensor, ...]:
         lambda_1, lambda_2 = args
         if self.fix_lambda_2:
-            lambda_2 = self.lb_obs * F.ones_like(lambda_2)
+            lambda_2 = -self.lb_obs * F.ones_like(lambda_2)
         else:
             # This makes sure that :math:`z +  \lambda_2 > 0`, where :math:`z > lb_obs`
-            lambda_2 = softplus(F, lambda_2) - self.lb_obs * F.ones_like(
+            lambda_2 = softplus(F, lambda_2) + self.lb_obs * F.ones_like(
```

Comment:
I think it should be `-` here. For example if observations are negative say with minimum value -100, then `lambda_2` with your change becomes `x - 100` where `x > 0`. Then the expression `z + \lambda2` is not guaranteed to be positive anymore and the transformation fails because it applies logarithm to this expression (when `\lambda_1 = 0`).

Reply:
Yes, thanks. Only the fixed case had the bug.